### ISSUE TITLE
prosessingset plot functions

### ISSUE THREAD

**Issue by kswang1029** [casangi/xradio#320]
currently there are only two plot functions (plot_phase_centers(), plot_antenna_positions()) associated with a processing set. Is there a plan to add more plot functions (for example plot uv coverage, plot passband) in the future? Or these are just a demonstration what could be plotted?

**Comment by Jan-Willem** [comment]
Yes, this was only intended to demonstrate the type of plotting we could offer. This will be expanded upon in the future. Plotting associated with metadata (everything that is not a data variable in the correlated_xds) can be implemented as a member function of the ps. For data-related plotting, such as UV coverage or visibilities, separate tasks will need to be created that leverage more advanced parallelization infrastructure.

**Comment by FedeMPouzols** [comment]
I think this is clarified. The MSv4 Review Committee stated in their report about this and related issues that: 
```
There were some concerns (315,318,319,320) about the limited feature set of xradio’s processing_set, but
this will grow with time, especially in the plotting area and the committee has no further concern.
```
And this RID was considered closed.

Please re-open or open a related issue if there is anything else to discuss or fix.
